tests/test_itemized.py

```diff
@@ -372,37 +372,6 @@ def test_image_number_range(self):
         self.assertTrue(all('2' <= each['image_number'] <= '3' for each in results))
 
 
-    def test_filter_individual_sched_a(self):
-        individuals = [
-            factories.ScheduleAFactory(receipt_type='15J'),
-            factories.ScheduleAFactory(line_number='12', contribution_receipt_amount=150),
-        ]
-        earmarks = [
-            factories.ScheduleAFactory(),
-            factories.ScheduleAFactory(
-                line_number='12',
-                contribution_receipt_amount=150,
-                memo_text='earmark',
-                memo_code='X',
-            ),
-        ]
-
-        is_individual = sa.func.is_individual(
-            ScheduleA.contribution_receipt_amount,
-            ScheduleA.receipt_type,
-            ScheduleA.line_number,
-            ScheduleA.memo_code,
-            ScheduleA.memo_text,
-            ScheduleA.contributor_id,
-            ScheduleA.committee_id,
-        )
-
-        rows = ScheduleA.query.all()
-        self.assertEqual(rows, individuals + earmarks)
-
-        rows = ScheduleA.query.filter(is_individual).all()
-        self.assertEqual(rows, individuals)
-
```

Comment:
By removing these from the `itemized` tests and moving them to the `integration` tests, does this help speed up the overall test execution and/or cut down on weird behaviors due to underlying test dependencies?  I don't think we had many, if at all, but was always wary of these things being spread out.

Reply:
Ideally all the tests in `test_itemized.py` should be moved into `test_integration`, or better still, into a directory like `tests/integration`. These tests use the test database, but bring their own schema by creating it from SQLAlchemy models. Instead they should be refactored to use the schema built using `flyway`. This was too much work to undertake.

I took the shortcut of only moving one test, `test_filter_individual_sched_a` as it uses the SQL function `is_individual`.

Reply:
Oh good point, we've discussed this before.  Yes, that would be far too much work for in here, but a nice small start to help push it forward!